src/measure_theory/bochner_integration.lean

```diff
@@ -305,6 +338,12 @@ begin
   exact integrable_neg hg
 end
 
+-- Typeclass inference has difficulty finding `has_scalar ℝ β` where `β` is a `normed_space` on `ℝ`
```

Comment:
Why don't you put this once and for all at the beginning of the file, to avoid repeating it in each section?